app/src/main/java/com/ichi2/apisample/helper/PcmToWavUtil.java

```diff
@@ -0,0 +1,134 @@
+package com.ichi2.apisample.helper;
+
+
+import android.media.AudioFormat;
+import android.media.AudioRecord;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class PcmToWavUtil {
+
+    /**
+     * Cached audio size
+     */
+    private int mBufferSize;
+    /**
+     * Sampling Rate
+     */
+    private int mSampleRate;
+    /**
+     * Number of channels
+     */
+    private int mChannel;
+
+
+    /**
+     * @param sampleRate sample rate, sampling rate
+     * @param channel    channel, channel
+     * @param encoding   Audio data format, audio format
+     */
+    public PcmToWavUtil(int sampleRate, int channel, int encoding) {
+        this.mSampleRate = sampleRate;
+        this.mChannel = channel;
+        this.mBufferSize = AudioRecord.getMinBufferSize(mSampleRate, mChannel, encoding);
+    }
+
+
+    /**
+     * pcm file to wav file
+     *
+     * @param inFilename  source file path
+     * @param outFilename target file path
+     */
+    public void pcmToWav(String inFilename, String outFilename) {
```

Comment:
the method's name is almost the same as the class's name, should be better to rename it to something like "convert"

Reply:
Done